dotnet/packages/Microsoft.TeamsAI/Microsoft.TeamsAI.Tests/IntegrationTests/AzureOpenAIPlannerTests.cs

```diff
@@ -148,7 +148,7 @@ public async Task AzureOpenAIPlanner_CompletePromptAsync_Unauthorized()
             var exception = await Assert.ThrowsAsync<TeamsAIException>(async () => await planner.CompletePromptAsync(turnContextMock.Object, turnStateMock.Object, promptTemplate, aiOptions));
 
             // Assert
-            Assert.Equal("Failed to perform AI prompt completion: Access denied: The request is not authorized, HTTP status: 401", exception.Message);
+            Assert.StartsWith("Error while executing AI prompt completion: Access denied due to invalid subscription key or wrong API endpoint. Make sure to provide a valid key for an active subscription and use a correct regional API endpoint for your resource.", exception.Message);
```

Comment:
The error message is now:
```
Access denied due to invalid subscription key or wrong API endpoint. Make sure to provide a valid key for an active subscription and use a correct regional API endpoint for your resource.
    Status: 401 (PermissionDenied)
    ErrorCode: 401
    
    Content:
    {"error":{"code":"401","message":"Access denied due to invalid subscription key or wrong API endpoint. Make sure to provide a valid key for an active subscription and use a correct regional API endpoint for your resource."}}
    
    Headers:
    apim-request-id: REDACTED
    Date: Mon, 11 Sep 2023 03:09:37 GMT
    Content-Length: 224
    Content-Type: application/json
```
So use `StartsWith` instead of `Equal`.